build.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 yarn install --immutable
-yarn workspaces foreach --exclude "@kaoto/step-extension-repository" --parallel --verbose run clean
-yarn workspaces foreach --exclude "@kaoto/step-extension-repository" --parallel --verbose run build
+yarn workspaces foreach --exclude "@kaoto/step-extension-repository" --exclude common --parallel --verbose run clean
```

Comment:
I think that this can stay as it was because we want to `clear` and `build`. That's ok :)
``` suggestion
yarn workspaces foreach --exclude "@kaoto/step-extension-repository" --parallel --verbose run clean
```

Reply:
ok, removed the `common` exclusion